### PR TITLE
Update Golang version to v1.19 for branch main

### PR DESCRIPTION
**Pull request by blackpiglet**
Thank you for contributing to Velero!

# Please add a summary of your change

# Does your change fix a particular issue?

Fixes #(issue)

# Please indicate you've done the following:

- [ ] [Accepted the DCO](https://velero.io/docs/v1.5/code-standards/#dco-sign-off). Commits without the DCO will delay acceptance.
- [ ] [Created a changelog file](https://velero.io/docs/v1.5/code-standards/#adding-a-changelog) or added `/kind changelog-not-required` as a comment on this pull request.
- [ ] Updated the corresponding documentation in `site/content/docs/main`.

### CODE REVIEW

**Comment by blackpiglet** [comment]
Still cannot workaround the VSC error without adding `-buildvcs=false`flag into code-generator's script.
```
error obtaining VCS status: exit status 128
	Use -buildvcs=false to disable VCS stamping.
```
Create an issue in code-generator repository to ask for some help.
https://github.com/kubernetes/code-generator/issues/156

**Review by Lyndon-Li — APPROVED** [review]

File: test/e2e/util/k8s/secret.go [hunk 1]
```diff
@@ -72,3 +72,25 @@ func WaitForSecretsComplete(c clientset.Interface, ns, secretName string) error
 func GetSecret(c clientset.Interface, ns, secretName string) (*v1.Secret, error) {
 	return c.CoreV1().Secrets(ns).Get(context.TODO(), secretName, metav1.GetOptions{})
 }
+
+// CreateVCCredentialSecret refer to https://github.com/vmware-tanzu/velero-plugin-for-vsphere/blob/v1.3.0/docs/vanilla.md
```
Comment (reasonerjt) [inline test/e2e/util/k8s/secret.go:76]: Is it related to golang bump up, how is it called?

**Review by reasonerjt — DISMISSED** [review]
LGTM

File: test/e2e/util/k8s/secret.go (hunk 1 above)
Comment (blackpiglet) [reply test/e2e/util/k8s/secret.go]: No. This is not needed, and it's introduced in resolving conflict. Removed.

**Review by reasonerjt — APPROVED** [review]

**Review by Lyndon-Li — APPROVED** [review]

**Pull request merged**